### DISCUSSION TITLE
lesson 11

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/foundry-full-course-cu#1028] [Q&A] [answered]
Hi , are _isApprovedOrOwner function still working? ( in MoodNft Contract )
for me not find in ERC721 and receive compiler error

**Comment by cromewar** [comment, marked as answer]
Yes you are right, please refer to: https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md#lesson-11
